website/views/layouts/layout.ejs

```diff
@@ -454,5 +483,6 @@
     });
     </script>
 
+
```

Comment:
Minor nitpick: can you revert this whitespace change?

Reply:
Done